build-info/dotnet/roslyn/netcore1.0/Latest_Packages.txt

```diff
@@ -6,3 +6,4 @@ Microsoft.CodeAnalysis.VisualBasic 1.3.0-beta1-20160606-02
 Microsoft.CodeAnalysis.VisualBasic.Workspaces 1.3.0-beta1-20160606-02
 Microsoft.CodeAnalysis.Compilers 1.3.0-beta1-20160606-02
 Microsoft.Net.Compilers.netcore 1.3.0-beta1-20160606-02
+Microsoft.DiaSymReader.PortablePdb 1.1.0-beta1-60606-02
```

Comment:
This build suffix doesn't match the rest - `beta1-20160606-02`.  Is that expected?